2-control-flow/sals-shipping/shipping.py

```diff
@@ -1,36 +1,37 @@
-# Sal's Shipping
-# Sonny Li
-
-weight = 80
-
-# Ground Shipping 🚚
-
-if weight <= 2:
-  cost_ground = weight * 1.5 + 20
-elif weight <= 6:
-  cost_ground = weight * 3.00 + 20
-elif weight <= 10:
-  cost_ground = weight * 4.00 + 20
-else:
-  cost_ground = weight * 4.75 + 20
-
-print("Ground Shipping $", cost_ground)
-      
-# Ground Shipping Premimum 🚚💨
-
-cost_ground_premium = 125.00
-
-print("Ground Shipping Premimium $", cost_ground_premium)
-
-# Drone Shipping 🛸
-
-if weight <= 2:
-  cost_drone = weight * 4.5
-elif weight <= 6:
-  cost_drone = weight * 9.00
-elif weight <= 10:
-  cost_drone = weight * 12.00
+weight = 41.5
+#ground shipping
+if(weight<2):
+  g_price = weight * 1.50 + 20.00
+  print("Ground shipping price is : " + str(g_price))
+elif(weight>2 and weight <= 6):
```

Comment:
we don't need to check `weight > 2` here as the conditional statement would terminate at the `if(weight<2)` step. This is why we use `elif` instead of a new, independent `if` statement. With `elif` we don't need to re-evaluate the previous condition.